src/sidebar/components/mentions/MentionPopoverContent.tsx

```diff
@@ -1,49 +1,83 @@
  */
 export default function MentionPopoverContent({
   content,
+  mentionMode,
 }: MentionPopoverContent) {
-  if (typeof content === 'string') {
-    return (
-      <>
-        No user with username <span className="font-bold">{content}</span>{' '}
-        exists
-      </>
-    );
-  }
+  const isUsernameMode = mentionMode === 'username';
 
   return (
-    <div className="flex flex-col gap-y-4">
-      <div className="flex flex-col gap-y-1.5">
-        <div data-testid="username" className="text-md font-bold">
-          @{content.username}
-        </div>
-        {content.display_name && (
-          <div data-testid="display-name" className="text-color-text-light">
-            {content.display_name}
+    <>
+      <div className="px-3 py-2">
+        {typeof content === 'string' ? (
+          <div data-testid="user-not-found">
+            No user with username <span className="font-bold">{content}</span>{' '}
+            exists
+          </div>
+        ) : (
+          <div className="flex flex-col gap-y-4">
+            <div className="flex flex-col gap-y-1.5">
+              {isUsernameMode && (
+                <div data-testid="username" className="text-md font-bold">
+                  @{content.username}
+                </div>
+              )}
+              {content.display_name && (
+                <div
+                  data-testid="display-name"
+                  className={classnames({
+                    'text-color-text-light': isUsernameMode,
+                    'text-md font-bold': !isUsernameMode,
+                  })}
+                >
+                  {content.display_name}
+                </div>
+              )}
+            </div>
+            {content.description && (
+              <div data-testid="description" className="line-clamp-2">
+                {content.description}
+              </div>
+            )}
+            {content.joined && (
+              <div data-testid="joined" className="text-color-text-light">
+                Joined{' '}
+                <b>{formatDateTime(content.joined, { includeTime: false })}</b>
+              </div>
+            )}
           </div>
         )}
       </div>
-      {content.description && (
-        <div data-testid="description" className="line-clamp-2">
-          {content.description}
-        </div>
-      )}
-      {content.joined && (
-        <div data-testid="joined" className="text-color-text-light">
-          Joined <b>{formatDateTime(content.joined, { includeTime: false })}</b>
-        </div>
-      )}
-    </div>
+      <div className="px-3 py-2 border-t">
+        <Link
+          variant="text-light"
+          classes="flex gap-x-1 items-center"
+          target="_blank"
+          href={
+            isUsernameMode
+              ? 'https://web.hypothes.is/help/mentions-for-the-hypothesis-web-app/'
+              : 'https://web.hypothes.is/help/mentions-for-the-hypothesis-lms-app/'
```

Comment:
This link assumes that any authority supporting mentions, and where usernames are not used, is the LMS app. It is initially true that the LMS app will be the only third-party using mentions, but eventually I expect other third parties will use them as well. We do have a mechanism in various other places that we use to allow the authority owner to provide custom links. We should eventually extend that to allow them to provide appropriate help links.